markup/house-style.md

```diff
@@ -75,6 +76,55 @@ We *don't* do this:
 ```html
 <meta charset="UTF-8" />
 ```
+### Form submit controls
+
+On modern browsers, `button type="submit"` provides a superset of functionality when compared to `input type=submit`.
+
+`button` elements are much easier to style than `input` elements. You can add
+inner HTML content (think `<em>`, `<strong>`, or even `<img>`), and use
+`::after` and `::before` pseudo-elements to achieve complex rendering while
+`input` only accepts a textual value attribute.
+
+Make sure to specify `name` and `value` attributes to maximize browsers
+compatibility.
+
+We do this:
+
+```html
+<form>
+	<button type="submit" name="myButton" value="foo">Continue</button>
+</form>
+```
+
+or
+
+We *don't* do this:
+
+```html
+<form>
+	<input type="submit" value="Continue">
+</form>
+```
+
+**Caution**: There is a shortcoming for Internet Explorer when using the `button`
+element. Indeed it submits the text within the `button` element as its value in
+the form data rather than the value of its `value` attribute.
+This becomes problematic when using multiple submit buttons in a single form with each a different value and purpose. 
+
+Here are some suggestions to work around these:
+
+- Interfaces could be redesigned so multiple submit buttons are not required.
+  - Replacing them with radio buttons and a single submit button (requires an
```

Comment:
```suggestion
  - Replace them with radio buttons and a single submit button (requires an
```

Reply:
There are funky line breaks in the markdown so I can't make the suggestion in the Github interface, but I would compress the first two sentences to this:

>**Caution**: In Internet Explorer,  the text within the `button` element is submitted as the value in the form data, rather than the value of the `value` attribute.

Also, I've already forgotten the details 😆 but is this just _some_ versions of IE? Might be worth mentioning that it's not all versions if that's the case